### ISSUE TITLE
Exclude BoneCP from the project

### ISSUE THREAD

**Issue by satybald** [playframework/play-slick#224]
Recently, we have a lot of problems with BoneCP. So we deciced to move to Hikaricp. I'm wondering how to define to not use BoneCP with Slick-Plugin?

I've define following configuration but still in my dependencies BoneCP
```
  "com.edulify" %% "play-hikaricp" % "1.5.0",
  "com.typesafe.play" %% "play-slick" % "0.8.0" exclude("com.jolbox","bonecp"),
```

**Comment by pvlugter** [comment]
Have you looked at the instructions for using the Hikari plugin? This describes how to disable the BoneCP plugin and enable HikariCP:

https://github.com/edulify/play-hikaricp.edulify.com#how-to-use

**Comment by dotta** [comment]
This is fixed in play-slick 1.0.0, since hikaricp is the default (and currently the only supported) connection pool :-)
